extensions/amp-accordion/amp-accordion.md

```diff
@@ -79,32 +79,32 @@ The events below will be triggered on `section`s of `accordion`.
 
 <table>
   <tr>
-    <td width="40%"><strong>`expand`</strong></td>
-    <td>This event is triggered on the target `section` that changes from collapsed state to expanded state. Notice that calling `expand` on an already expanded `section` would not trigger this event.</td>
+    <td width="40%"><strong><code>expand</code></strong></td>
+    <td>This event is triggered on the target <code>section</code> that changes from collapsed state to expanded state. Notice that calling <code>expand</code> on an already expanded <code>section</code> would not trigger this event.</td>
   </tr>
   <tr>
-    <td width="40%"><strong><strong>`collapse`</strong></td>
-    <td>This event is triggered on the target `section` that changes from expanded state to collapsed state. Notice that calling `collapse` on an already collapsed `section` would not trigger this event.</td>
+    <td width="40%"><strong><strong><code>collapse</code></strong></td>
+    <td>This event is triggered on the target <code>section</code> that changes from expanded state to collapsed state. Notice that calling <code>collapse</code> on an already collapsed <code>section</code> would not trigger this event.</td>
   </tr>
 </table>
 
 ### Actions
 <table>
   <tr>
-    <td width="40%"><strong>`expand`</strong></td>
-    <td>This event is triggered on the target `section` that changes from collapsed state to expanded state. Notice that calling `expand` on an already expanded `section` would not trigger this event.</td>
+    <td width="40%"><strong><code>expand</code></strong></td>
+    <td>This event is triggered on the target <code>section</code> that changes from collapsed state to expanded state. Notice that calling <code>expand</code> on an already expanded <code>section</code> would not trigger this event.</td>
   </tr>
   <tr>
-    <td width="40%"><strong>`toggle`</strong></td>
-    <td>This action toggles between the `expanded` and `collapsed` states of the `amp-accordion`. When called with no arguments, it will toggle all sections of the accordion. A single section may be specified with the `section` argument and the corresponding `id` as the value.</td>
+    <td width="40%"><strong><code>toggle</code></strong></td>
+    <td>This action toggles between the <code>expanded</code> and <code>collapsed</code> states of the <code>amp-accordion<code/>. When called with no arguments, it will toggle all sections of the accordion. A single section may be specified with the <code>section</code> argument and the corresponding <code>id</code> as the value.</td>
```

Comment:
```suggestion
    <td>This action toggles between the <code>expanded</code> and <code>collapsed</code> states of the <code>amp-accordion</code>. When called with no arguments, it will toggle all sections of the accordion. A single section may be specified with the <code>section</code> argument and the corresponding <code>id</code> as the value.</td>
```